raiden-ts/tests/integration/epics/mediate.spec.ts

```diff
@@ -292,3 +292,22 @@ test('flatFee', () => {
 
   expect(flatFee.schedule(config, null as any)).toEqual({ flat: config });
 });
+
+test('proportionalFee', () => {
+  expect(proportionalFee.name).toEqual('proportional');
+  expect(proportionalFee.emptySchedule).toEqual({ proportional: Zero });
+
+  const config = proportionalFee.decodeConfig('10000'); // 1% = 0.01*1e6
+  expect(config).toEqual(BigNumber.from(4975)); // perChannel = perHop / (perHop + 2)
+  expect(
+    proportionalFee
+      .fee(
+        config,
+        null as any,
+        null as any,
+      )(decode(UInt(32), 10213))
+      .toNumber(),
+  ).toBeWithin(100 - 1, 100 + 2); // from SP = fee = 100±1
```

Comment:
Could you add some more test cases here. Ideally all of them listed here: https://github.com/raiden-network/raiden/blob/6f36e3c57eb81d5355af03d1928c41da8ea02ffd/raiden/tests/unit/transfer/mediated_transfer/test_mediation_fee.py#L246-L271

Reply:
Since this is related only to proportional fees, would it be ok if I add only the pure respective tests from there? The tests integration multiple models will be added when we add the imbalance fees, since those will require some tweaking on our algorithm.

Reply:
Sure, that's fine.

But in that test it's only fixed and proportional fees, so we should already be able to implement those as well.

Reply:
ok, added the proportional tests only, had to adjust the algorith a little, but it´s working fine now: :heavy_check_mark: